requirements.txt

```diff
@@ -4,3 +4,10 @@ wandb>=0.10.32,<1  # pip will likely update it to 0.12.1, but it is probably ok
 transformers>=4.6.0,<5  # pip will likely update it to 4.10.0, but it is probably ok and good for bugfixes.
```

Comment:
The `torch` version used in the `requirements.txt` is `1.8.1` which seems to have some issue with the `flair` library I am using (https://github.com/flairNLP/flair/issues/2137). Can we use torch==1.9.0 instead?

Reply:
We can definitely test this change (in another PR). The last version available on jz seems to be 1.9.0. :)